docs/answers.md

```diff
@@ -38,6 +38,58 @@ const annotateData = (array) => array.reduce(reducer, [])
 const newData = annotateData(data)
 ```
 
+## Objects and Classes
+
+### Delays
+
+```{js}
+class Delay {
+  constructor(initialValue){
+    this.nextValue = initialValue
+  }
+
+  call(nextValue) {
+    let previousValue = this.nextValue
+    this.nextValue = nextValue
+    return previousValue
+  }
+}
+```
+
+### Filtering
+
+```{js}
+class Filter {
+  constructor(...values){
+    this.filterValues = values
+  }
+
+  call(inputValue){
+    return this.filterValues.some((value) => value === inputValue) ? null : inputValue
```

Comment:
It is probably more efficient to use `indexOf`:
```
return this.filterValues.indexOf(value) === -1 ? inputValue : null
```

Reply:
Hi @gvwilson, 
I performed some tests out of curiosity to learn more about your suggestion. 
I did run a [performance test](https://jsperf.com/some-n-indexof ). The results showed me that both are comparatively same.
Please advise.